### PR TITLE
Loader docs

### PR DESCRIPTION
**Pull request by williamFalcon**
update dataloader docs

### CODE REVIEW

File: docs/source/introduction_guide.rst [hunk 1]
```diff
@@ -28,6 +28,22 @@ to use inheritance to very quickly create an AutoEncoder.
...
+
+.. code-block::
+
+    $ conda activate my_env
+    $ pip install pytorch-lightning
+
+Or without conda environments, anywhere you can use pip.
+
+.. code-block::
+    $ pip install pytorch-lightning
```
Comment (Borda) [inline docs/source/introduction_guide.rst:43]:
```suggestion
.. code-block::bash

    conda activate my_env
    pip install pytorch-lightning

Or without conda environments, anywhere you can use pip.

.. code-block::bash

    pip install pytorch-lightning
```

File: docs/source/multiple_loaders.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
+Multiple Datasets
+=================
+Lightning supports multiple dataloaders in a few ways.
+
+1. Create a dataloader that iterates both datasets under the hood.
+2. In the validation and test loop you also have the option to return multiple dataloaders
+which lightning will call sequentially.
```
Comment (Borda) [inline docs/source/multiple_loaders.rst:7]:
```suggestion
1. Create a dataloader that iterates both datasets under the hood.
2. In the validation and test loop you also have the option to return multiple dataloaders
    which lightning will call sequentially.
```

**Comment by williamFalcon** [comment]
@awaelchli any thoughts on this? seems to be the sphinx extension.

image

**Comment by awaelchli** [comment]
could you run `pip install -r docs/requirements.txt` just to be sure because CI doesn't seem to complain.

**Comment by Borda** [comment]
also, there can be some forgetting files in source folder which were generated earlier (with apidoc) and now ar missing in the VCS...

**Comment by williamFalcon** [comment]
ok, that fixed it.
But now we have these guys at the bottom. They shouldn't be there

image

**Comment by awaelchli** [comment]
@williamFalcon See PyTorchLightning/lightning_sphinx_theme#10 for a fix for the navigation items

**Pull request merged**